crates/nargo_cli/build.rs

```diff
@@ -54,15 +54,15 @@ fn load_conf(conf_path: &Path) -> BTreeMap<String, Vec<String>> {
     conf_data
 }
 
-fn generate_tests(test_file: &mut File, experimental_ssa: bool) {
+fn generate_tests(test_file: &mut File, legacy_ssa: bool) {
     // Try to find the directory that Cargo sets when it is running; otherwise fallback to assuming the CWD
     // is the root of the repository and append the crate path
     let manifest_dir = match std::env::var("CARGO_MANIFEST_DIR") {
         Ok(dir) => PathBuf::from(dir),
         Err(_) => std::env::current_dir().unwrap().join("crates").join("nargo_cli"),
     };
-    // Choose the test directory depending on whether we are in the SSA refactor module or not
-    let test_sub_dir = if experimental_ssa { "test_data_ssa_refactor" } else { "test_data" };
+    // Choose the test directory depending on whether we are in the legacy SSA module or not
+    let test_sub_dir = if legacy_ssa { "test_data" } else { "test_data_ssa_refactor" };
```

Comment:
Can we move the new tests into `test_data` and move the legacy tests into a new folder like `test_data_legacy` instead?

Reply:
I wanted to avoid moving around a lot of the various files as we can address this in future PRs as we remove the legacy SSA.